[2021-09-27 15:58:46,812] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:58:39.530261+00:00 [queued]>
[2021-09-27 15:58:46,842] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:58:39.530261+00:00 [queued]>
[2021-09-27 15:58:46,845] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:58:46,847] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:58:46,849] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:58:46,890] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): echo_hello> on 2021-09-27T15:58:39.530261+00:00
[2021-09-27 15:58:46,899] {standard_task_runner.py:52} INFO - Started process 38480 to run task
[2021-09-27 15:58:46,916] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'day1_challenge2', 'echo_hello', '2021-09-27T15:58:39.530261+00:00', '--job-id', '108', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/day1_challenge2.py', '--cfg-path', '/tmp/tmps9bz2z2n', '--error-file', '/tmp/tmpalz0r8yc']
[2021-09-27 15:58:46,927] {standard_task_runner.py:77} INFO - Job 108: Subtask echo_hello
[2021-09-27 15:58:47,205] {logging_mixin.py:109} INFO - Running <TaskInstance: day1_challenge2.echo_hello 2021-09-27T15:58:39.530261+00:00 [running]> on host 196580edc338
[2021-09-27 15:58:49,320] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Attasil Chitwirote
AIRFLOW_CTX_DAG_ID=day1_challenge2
AIRFLOW_CTX_TASK_ID=echo_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:58:39.530261+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T15:58:39.530261+00:00
[2021-09-27 15:58:49,405] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 15:58:49,435] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo 'hello'"]
[2021-09-27 15:58:49,717] {subprocess.py:74} INFO - Output:
[2021-09-27 15:58:49,777] {subprocess.py:78} INFO - hello
[2021-09-27 15:58:49,783] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 15:58:50,251] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=day1_challenge2, task_id=echo_hello, execution_date=20210927T155839, start_date=20210927T155846, end_date=20210927T155850
[2021-09-27 15:58:50,440] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:58:50,719] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
